plugins/PolylangPro.php

```diff
@@ -37,17 +37,20 @@ public function __construct( $version = '' ) {
 	 */
 	public function getDownloadUrl() {
 		$http     = new Http();
-		$response = json_decode( $http->post( 'https://polylang.pro', array(
+		$name = 'Polylang Pro';
```

Comment:
@jessedyck Can you undo the changes in this file? Nobody has faced any issues with Polylang, and I don't want to inadvertently cause any issues. WP All Import / Export seem to have a weird EDD implementation.